vcpkg.json

```diff
@@ -13,6 +13,7 @@
     "boost-program-options",
     "boost-coroutine2",
     "boost-stacktrace",
+    "boost-uuid",
```

Comment:
where is this used?

Reply:
this was used, but when building without this everything still builds, removing this.